runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -456,12 +454,19 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 					}
 				}
 				break;
+				
+			default :
+				// unreachable since we currently do not expect anything other than primitive arrays to be using them.
+				{
+					Assert_MM_unreachable();
```

Comment:
To make this correct you need to add `#include "ModronAssertions.h"` at the beginning of the file

Reply:
Yea, I realized that when I was testing my changes. I'm currently rebuilding my changes to see if i missed anything else